libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/helpers/WPHtmlTagHandler.java

```diff
@@ -33,27 +34,30 @@ public void handleTag(final boolean opening, final String tag, Editable output,
     }
 
     private void handleListTag(Editable output) {
-        if (mListParents.lastElement().equals("ul")) {
-            output.append("\n");
-            String[] split = output.toString().split("\n");
-            int start = 0;
-            if (split.length != 1) {
-                int lastIndex = split.length - 1;
-                start = output.length() - split[lastIndex].length() - 1;
+        int size = mListParents.size();
+        if (size > 0) {
+            if (mListParents.get(size - 1).equals("ul")) {
+                output.append("\n");
+                String[] split = output.toString().split("\n");
+                int start = 0;
+                if (split.length != 1) {
+                    int lastIndex = split.length - 1;
+                    start = output.length() - split[lastIndex].length() - 1;
+                }
+                output.setSpan(new BulletSpan(15 * mListParents.size()), start, output.length(), 0);
+            } else if (mListParents.get(size - 1).equals("ol")) {
+                mListItemCount++;
+                output.append("\n");
+                String[] split = output.toString().split("\n");
+                int start = 0;
+                if (split.length != 1) {
+                    int lastIndex = split.length - 1;
+                    start = output.length() - split[lastIndex].length() - 1;
+                }
+                output.insert(start, mListItemCount + ". ");
+                output.setSpan(new LeadingMarginSpan.Standard(15 * mListParents.size()), start,
```

Comment:
Let's move `15` to a constant as it's being used at least twice in code. Let's try to find a proper name for such constant, that describes what it is